docs/source/conf.py

```diff
@@ -39,19 +40,60 @@
 #
 # needs_sphinx = '1.0'
 
-# Add any Sphinx extension module names here, as strings. They can be
+# Add any Sphinx extension module names here as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    "sphinx.ext.intersphinx",
     "sphinx.ext.autodoc",
-    "sphinx_gallery.gen_gallery",
     "sphinx.ext.todo",
-    "sphinx.ext.napoleon",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.coverage",
+    "sphinx_copybutton",
+    "sphinx_gallery.gen_gallery",
+    "numpydoc",
     "pydata_sphinx_theme",
     "nbsphinx",
 ]
 
-autosummary_generate = True
+
+# Intersphinx mapping
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/dev", None),
+    "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
+    "numpy": ("https://numpy.org/devdocs", None),
+    "matplotlib": ("https://matplotlib.org/stable", None),
+    "imageio": ("https://imageio.readthedocs.io/en/stable", None),
+    "pandas": ("https://pandas.pydata.org/pandas-docs/stable", None),
+    "pytest": ("https://docs.pytest.org/en/stable", None),
```

Comment:
@PipKat Where are the link to pandas, scipy and imageio used? If it's not used, I'm not sure we need to add them, it might make think that we depend on those modules

Reply:
@cbellot000 Links to pandas, scipy and imageio are probably not used. We just referenced the same ones that were in AEDT and MAPDL. We can remove them. As far as getting the documentation pipeline to work, I have no idea how to make this happen. Pinging @akaszynski for advice.